massa-pool/src/error.rs

```diff
@@ -11,11 +11,17 @@ pub enum PoolError {
     /// there was an inconsistency between containers
     ContainerInconsistency(String),
     /// Protocol error {0}
-    ProtocolError(#[from] ProtocolError),
+    ProtocolError(#[from] Box<ProtocolError>),
     /// channel error : {0}
     ChannelError(String),
     /// Join error {0}
     JoinError(#[from] tokio::task::JoinError),
     /// models error: {0}
     ModelsError(#[from] ModelsError),
 }
+
+impl From<ProtocolError> for PoolError {
+    fn from(protocol_error: ProtocolError) -> Self {
+        PoolError::ProtocolError(Box::new(protocol_error))
+    }
+}
```

Comment:
What's the purpose of that ?

Reply:
I guess helping implicit error conversion (the power of `?` operator and `anyhow::Error` which is behind the hood a `Box<dyn std::error::Error>`)!